<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mkmk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pB_Create">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Makefile">
     <attribute name="title">
      <string>Makefile</string>
     </attribute>
     <widget class="QGroupBox" name="gB_Language">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Language</string>
      </property>
      <widget class="QWidget" name="layoutWidget_A">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_Language">
        <item>
         <widget class="QRadioButton" name="rB_CPP">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>C++</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rB_C">
          <property name="text">
           <string>C</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rB_65x02">
          <property name="text">
           <string>65x02</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gB_Binary">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Binary</string>
      </property>
      <widget class="QWidget" name="layoutWidget_B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>108</width>
         <height>58</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_Binary">
        <item>
         <widget class="QRadioButton" name="rB_Executable">
          <property name="text">
           <string>Executable</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rB_Library">
          <property name="text">
           <string>Library</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gB_Environment">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Environment</string>
      </property>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_Environment">
        <item>
         <widget class="QRadioButton" name="rB_Production">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Production</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rB_Release">
          <property name="text">
           <string>Release</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rB_Debug">
          <property name="text">
           <string>Debug</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Extras">
     <attribute name="title">
      <string>Extras</string>
     </attribute>
     <widget class="QGroupBox" name="gB_SDL2_Librairies">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>201</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extra libs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>SDL2 librairies</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>108</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_SDL2_Librairies">
        <item>
         <widget class="QCheckBox" name="cB_SDL2">
          <property name="text">
           <string>SDL2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cB_SDL2_image">
          <property name="text">
           <string>SDL2 image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cB_SDL2_mixer">
          <property name="text">
           <string>SDL2 mixer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cB_SDL2_net">
          <property name="text">
           <string>SDL2 net</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cB_SDL2_ttf">
          <property name="text">
           <string>SDL2 ttf</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="title">
       <string>Folders</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget_C">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>203</width>
         <height>166</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_Folders">
        <item>
         <layout class="QHBoxLayout" name="hL_BASEDIR">
          <item>
           <widget class="QLabel" name="l_BASEDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BASEDIR=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_BASEDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hL_SRCDIR">
          <item>
           <widget class="QLabel" name="l_SRCDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SRCDIR =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_SRCDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hL_OBJDIR">
          <item>
           <widget class="QLabel" name="l_OBJDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>OBJDIR =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_OBJDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hL_DEPDIR">
          <item>
           <widget class="QLabel" name="l_DEPDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>DEPDIR =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_DEPDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hL_BINDIR">
          <item>
           <widget class="QLabel" name="l_BINDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BINDIR =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_BINDIR">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gB_CompilerFlags">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Compiler Flags</string>
      </property>
      <widget class="QWidget" name="layoutWidget_D">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_CompilerFlags">
        <item>
         <layout class="QHBoxLayout" name="hL_DBG">
          <item>
           <widget class="QLabel" name="l_DBG">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>DBG =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_DBG">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hL_REL">
          <item>
           <widget class="QLabel" name="l_REL">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>REL =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_REL">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_PRD">
          <item>
           <widget class="QLabel" name="l_PRD">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>PRD =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_PRD">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gB_WarningFlags">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Warning Flags</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_WarningFlags">
        <item>
         <layout class="QHBoxLayout" name="hL_WRN">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="l_WRN">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>WRN =</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_WRN">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gB_ExtraFlags">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {background-color: #EEE; }
QLineEdit    { background-color: white; }</string>
      </property>
      <property name="title">
       <string>Extra Flags</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vL_XTRA">
        <item>
         <layout class="QHBoxLayout" name="hL_XTRA">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="l_XTRA">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>XTRA=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lE_XTRA">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>rB_C</tabstop>
  <tabstop>rB_CPP</tabstop>
  <tabstop>rB_Executable</tabstop>
  <tabstop>rB_Library</tabstop>
  <tabstop>lE_BASEDIR</tabstop>
  <tabstop>lE_SRCDIR</tabstop>
  <tabstop>lE_DEPDIR</tabstop>
  <tabstop>lE_OBJDIR</tabstop>
  <tabstop>lE_BINDIR</tabstop>
  <tabstop>lE_DBG</tabstop>
  <tabstop>lE_REL</tabstop>
  <tabstop>lE_PRD</tabstop>
  <tabstop>pB_Create</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
